libkineto/src/output_json.cpp

```diff
@@ -55,6 +55,7 @@ ChromeTraceLogger::ChromeTraceLogger(const std::string& traceFileName, int smCou
   openTraceFile(fileName_, traceOf_);
 #ifdef HAS_CUPTI
   smCount_ = CuptiActivityInterface::singleton().smCount();
```

Comment:
I assume sm count is also available via cudaOccDeviceProp? If so, let's move to that (in a separate PR - I can do it as well).

Reply:
Yes, we can reuse cudaOccDeviceProp.numSms. Let's do it in another PR.